addons/base/views.py

```diff
@@ -869,6 +869,14 @@ def addon_view_file(auth, node, file_node, version):
         args={'url': download_url.url}
     )
 
+    version_names = []
+    versions = file_node.versions.order_by('-created')
+    if versions.count():
+        for vers in versions:
```

Comment:
Any idea how much this will affect response times? Presumably imperceptibly for files with only a version or two, but what about 10, 100, etc?